libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/model/Maneuver.kt

```diff
@@ -47,7 +47,7 @@ class Maneuver internal constructor(
         result = 31 * result + (secondary?.hashCode() ?: 0)
         result = 31 * result + (sub?.hashCode() ?: 0)
         result = 31 * result + (laneGuidance?.hashCode() ?: 0)
-        result = 31 * result + (maneuverPoint?.hashCode() ?: 0)
+        result = 31 * result + maneuverPoint.hashCode()
```

Comment:
NIT: Actually the same change can be applied to other properties, because
```kotlin
public inline fun Any?.hashCode(): Int = this?.hashCode() ?: 0
```